src/main/kotlin/com/sourcegraph/cody/autocomplete/CodyAutocompleteManager.kt

```diff
@@ -172,7 +172,7 @@ class CodyAutocompleteManager {
 
     val textDocument: TextDocument = IntelliJTextDocument(editor, project)
 
-    if (isTriggeredExplicitly && CodyAuthenticationManager.getInstance().hasNoActiveAccount()) {
+    if (isTriggeredExplicitly && !CodyAccount.hasActiveAccount()) {
       HintManager.getInstance().showErrorHint(editor, "Cody: Sign in to use autocomplete")
```

Comment:
Ugh, we should link to the sign in panel here! Maybe add a TODO?

Reply:
Actually I think we should just show it. 
I will do the changes, thanks!